### PR TITLE
chore(suite): sentry upgrade 

### LINKED ISSUE #10079: [mobile] onUnhandled error
1. This user affects lot of users but I am not sure about implications (it probably doesn't crash app)
2. Source maps doesn't seems to work for Android so you will need to get APK of that version, extract it and look into raw JS code

```
{
message: Cannot read property 'txid' of null, 
name: TypeError, 
stack: 
TypeError: Cannot read property 'txid' of null
    at anonymous (address at index.android.bundle:1:6050330)
    at filter (native)
    at ?anon_0_ (address at index.android.bundle:1:6049533)
    at next (native)
    at asyncGeneratorStep (address at index.android.bundle:1:619818)
    at _next (address at index.android.bundle:1:620088)
    at anonymous (address at index.android.bundle:1:620040)
    at tryCallTwo (address at index.android.bundle:1:281288)
    at doResolve (address at index.android.bundle:1:282606)
    at Promise (address at index.android.bundle:1:281384)
    at anonymous (address at index.android.bundle:1:619961)
    at apply (native)
    at anonymous (address at index.android.bundle:1:6050406)
    at anonymous (address at index.android.bundle:1:4204918)
    at call (native)
    at step (address at index.android.bundle:1:2113050)
    at anonymous (address at index.android.bundle:1:2112163)
    at anonymous (address at index.android.bundle:1:4201048)
    at tryCallTwo (address at index.android.bundle:1:281288)
    at doResolve (address at index.android.bundle:1:282606)
    at Promise (address at index.android.bundle:1:281384)
    at w (address at index.android.bundle:1:4200960)
    at anonymous (address at index.android.bundle:1:4204196)
    at anonymous (address at index.android.bundle:1:4204077)
    at anonymous (address at index.android.bundle:1:4225541)
    at anonymous (address at index.android.bundle:1:6994585)
    at anonymous (address at index.android.bundle:1:7005489)
    at commitHookEffectListMount (address at index.android.bundle:1:432504)
    at flushPassiveEffects (address at index.android.bundle:1:446146)
    at anonymous (address at index.android.bundle:1:445266)
    at J (address at index.android.bundle:1:386607)
    at R (address at index.android.bundle:1:386941)
    at apply (native)
    at anonymous (address at index.android.bundle:1:293685)
    at _callTimer (address at index.android.bundle:1:292634)
    at _callReactNativeMicrotasksPass (address at index.android.bundle:1:292798)
    at callReactNativeMicrotasks (address at index.android.bundle:1:294767)
    at __callReactNativeMicrotasks (address at index.android.bundle:1:197888)
    at anonymous (address at index.android.bundle:1:196981)
    at __guard (address at index.android.bundle:1:197729)
    at flushedQueue (address at index.android.bundle:1:196892)
    at callFunctionReturnFlushedQueue (address at index.android.bundle:1:196748)
}
```

Android Sentry Issue: [SUITE-NATIVE-3E](https://satoshilabs.sentry.io/issues/4213629601/?referrer=github_integration)
iOS Sentry issue: [SUITE-NATIVE-2P](https://satoshilabs.sentry.io/issues/4185450328/?referrer=github_integration)

### PR DESCRIPTION
**Pull request by vytick**
Upgrade of library should help with unhandled promise errors based on https://github.com/getsentry/sentry-react-native/pull/3423

Resolves #10079

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5435,6 +5465,16 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@sentry/core@npm:7.81.1, @sentry/core@npm:^7.77.0":
```
Comment (juriczech) [inline yarn.lock:5468]:
```suggestion
"@sentry/core@npm:7.81.1:
```

File: yarn.lock [hunk 2]
```diff
@@ -5451,39 +5491,26 @@ __metadata:
...
+    "@sentry/types": "npm:7.81.1"
+    "@sentry/utils": "npm:7.81.1"
+  checksum: a0b65c8e0d76ce92ce02e5035b3b7f7563e97db4b5c1d1c43cc3050394e41ca82e95cdde652fb2f5d7b948c91eb83a9568da83866ad8adbfe3078932baf6c4fd
   languageName: node
   linkType: hard
 
-"@sentry/integrations@npm:^7.77.0":
-  version: 7.77.0
-  resolution: "@sentry/integrations@npm:7.77.0"
+"@sentry/integrations@npm:7.81.1, @sentry/integrations@npm:^7.77.0":
```
Comment (juriczech) [inline yarn.lock:5505]:
```suggestion
"@sentry/integrations@npm:7.81.1:
```

File: yarn.lock [hunk 3]
```diff
@@ -5373,44 +5358,89 @@ __metadata:
...
+  linkType: hard
+
+"@sentry/cli-win32-x64@npm:2.23.0":
+  version: 2.23.0
+  resolution: "@sentry/cli-win32-x64@npm:2.23.0"
+  conditions: os=win32 & cpu=x64
+  languageName: node
+  linkType: hard
+
+"@sentry/cli@npm:2.23.0, @sentry/cli@npm:^2.21.2":
```
Comment (juriczech) [inline yarn.lock:5410]:
```suggestion
"@sentry/cli@npm:2.23.0":
```

File: yarn.lock [hunk 4]
```diff
@@ -5344,16 +5329,16 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@sentry/browser@npm:^7.77.0":
-  version: 7.77.0
-  resolution: "@sentry/browser@npm:7.77.0"
+"@sentry/browser@npm:7.81.1, @sentry/browser@npm:^7.77.0":
```
Comment (juriczech) [inline yarn.lock:5332]:
```suggestion
"@sentry/browser@npm:7.81.1":
```

File: yarn.lock [hunk 5]
```diff
@@ -5620,6 +5625,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@sentry/utils@npm:7.81.1, @sentry/utils@npm:^7.60.0":
```
Comment (juriczech) [inline yarn.lock:5628]:
```suggestion
"@sentry/utils@npm:7.81.1":
```

**Review by juriczech — COMMENTED** [review]
Please update the dependencies in all packages so we have one version of these libs instead having multiple versions of same packages throughout the repo. Try running `yarn why LIBRARY_NAME` to reduce these versions to one if possible.

File: yarn.lock (hunk 5 above)
Comment (vytick) [reply yarn.lock:5628]: c786bf1554

File: yarn.lock (hunk 4 above)
Comment (vytick) [reply yarn.lock:5332]: c786bf1554

File: yarn.lock (hunk 3 above)
Comment (vytick) [reply yarn.lock:5410]: sentry cli **^2.21.2** is added as a dependency of `@sentry/bundler-plugin-core` and even the newest version of the package does not use **2.23.0**

File: yarn.lock (hunk 2 above)
Comment (vytick) [reply yarn.lock:5505]: https://github.com/trezor/trezor-suite/commit/c786bf1554c6f9e97a39b4a4988f639506b393eb

File: yarn.lock (hunk 1 above)
Comment (vytick) [reply yarn.lock:5468]: https://github.com/trezor/trezor-suite/commit/c786bf1554c6f9e97a39b4a4988f639506b393eb

**Review by juriczech — APPROVED** [review]

**Comment by matejkriz** [comment]
@SocketSecurity ignore @sentry/react-native@5.15.2

**Review by matejkriz — APPROVED** [review]
sentry in desktop app works fine 👌  https://satoshilabs.sentry.io/issues/4832433778/events/7daee8b3e9184941bf1306fa68f00ad0/

**Pull request merged**